docs/usage/configuration/headings.md

```diff
@@ -57,6 +57,42 @@ plugins:
 ////
 ///
 
+## `heading`
+
+- **:octicons-package-24: Type [`str`][] :material-equal: `""`{ title="default value" }**
+<!-- - **:octicons-project-template-24: Template :material-null:** (N/A) -->
+
+A custom string to use as the heading of the root of the object (i.e. the object specfied directly after the identifier `:::`). This will override the default heading generated by the plugin.
+
+!!! warning "Not advised to be used as a general configuration option"
+    This option is not advised to be used as a general configuration option, as it will override the default heading for all objects. It is recommended to use it only in specific cases where you want to override the heading for a specific object.
```

Comment:
We use mkdocs-callouts for admonitions/callouts :slightly_smiling_face: 

```suggestion
WARNING: **Not advised to be used as a global configuration option.** This option is not advised to be used as a global configuration option, as it will override the default heading for all objects. It is recommended to use it only in specific cases where you want to override the heading for a specific object.
```

Maybe we should actually ignore the global value. I'll keep that in mind, no need to add more logic to this PR :slightly_smiling_face: